src/components/externalLinks/externalLinks.component.scss

```diff
@@ -256,9 +256,20 @@
     }
 
     .external-link-chip {
-        max-width: 250px;
-        padding: 4px 6px;
-        margin: 4px 8px 4px 0;
+        grid-template-columns: auto 0;
```

Comment:
Please check if layout shift is happening across browsers, if happening please connect, we can improve the css by adding max-width

Reply:
working fine